content/reference/user/organization/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Organization"
+description: "Learn how to access organization details."
+date: 2021-08-04T14:29:04-00:00
+lastmod: 2021-08-04T14:29:04-00:00
+weight: 10081
+---
+
+To consult information regarding an organization, click on the **Organization name link**.
+
+{{< img src="organization-page-link.png" alt="Organization page link" >}}
+
+## Profile information
+
+{{< img src="organization-profile.png" alt="Organization profile information" >}}
+
+* **Avatar** - Compose by default from the two first characters of your **Organization name**.
+* **Organization Name** - Display name for your organization.
+* **Organization Slug** - Unique string name.
+
+
+{{< alert tip >}}
+Click on the pen icon to edit the **Organization name**.
+{{< /alert >}}
+
+## Credit consumption
+
+{{< img src="organization-credit.png" alt="Organization credit informations" >}}
+
+* **Blue** - Available credits for the current month.
+* **Orange** - Consumed credits during the current month.
+
+### Admin users
+
+Display all the System Admin in your organization.
+
+{{< img src="organization-admin-user.png" alt="Organization users admin" >}}
+
+For each user, you will find his GitHub username, first name, and last name.
+
+### Credits
+
+{{< alert warning >}}
+This section is only available for System Admin.
+{{< /alert >}}
+
+Credits consumption history.
+
+{{< img src="organization-credits-row.png" alt="Organization credits view in row" >}}
+
+Clicking on a row will show you all the credits consumption details for each month.
+
+{{< img src="organization-credits-detail.png" alt="Organization credits view in detail" >}}
+
+* **Type** - Give you the action name, a link redirect to the runs.
+* **Date** - Day when the action takes effect.
+* **Credits** - Number of credits gain or used by the action.
+
+### Plans
+
+{{< alert warning >}}
+This section is only available for System Admin.
```

Comment:
This section is only available to the system administrator.

Reply:
Done